producer/src/main/kotlin/streams/RoutingConfiguration.kt

```diff
@@ -175,17 +175,13 @@ data class RelationshipRoutingConfiguration(val name: String = "",
 
                         val newRecordBefore: RecordChange? = if (relationshipPayload.before != null) {
                             val recordBefore = relationshipPayload.before as RelationshipChange
-                            recordBefore.copy(properties = filterProperties(streamsEvent.payload.before, it),
-                                    start = recordBefore.start,
-                                    end = recordBefore.end)
+                            recordBefore.copy(properties = filterProperties(streamsEvent.payload.before, it))
```

Comment:
not sure if we should do copy here, keep it as a perf-improvement issue for later.